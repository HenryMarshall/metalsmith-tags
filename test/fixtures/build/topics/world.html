<h1>Posts for world</h1>
<ul>
	<li>test</li>
</ul>