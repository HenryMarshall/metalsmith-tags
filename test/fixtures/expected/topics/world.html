<h1>Posts for world</h1>
<ul>
	<li><small>19/07/2014</small> test</li>
</ul>